src/main/java/teammates/logic/backdoor/BackDoorLogic.java

```diff
@@ -153,8 +154,17 @@ public String persistDataBundle(DataBundle dataBundle)
         }
         frDb.createFeedbackResponses(responses.values());
 
+        Set<String> sessionStrings = new HashSet<>();
+        
         for (FeedbackResponseAttributes response : responses.values()) {
-            updateRespondants(response.feedbackSessionName, response.courseId);
+            
+            String sessionString = response.feedbackSessionName + " % " + response.courseId;
```

Comment:
@unyoungwax, it should be fine to not have the space between the %, to be consistent with <code>FeedbackSession.java</code>